lib/std/math/big/int.zig

```diff
@@ -1803,7 +1818,7 @@ pub const Mutable = struct {
         };
 
         // Handle 0-bit integers.
-        if (req_limbs == 0 or abs_trunc_a.eqlZero()) {
+        if (bit_count == 0 or abs_trunc_a.eqlZero()) {
```

Comment:
Note that this is redundant with #23209 if you end up needing to resolve conflicts.